cartography/driftdetect/model.py

```diff
@@ -1,72 +1,81 @@
 import json
 import logging
 from marshmallow import Schema, fields, post_load
-from enum import IntEnum
-
 
 logger = logging.getLogger(__name__)
 
 
-class DriftDetectorType(IntEnum):
-    EXPOSURE = 1
-
-
-class DriftDetector(object):
-    def __init__(self,
-                 name,
-                 validation_query,
-                 expectations,
-                 detector_type):
+class DriftState(object):
```

Comment:
Consider renaming to simply `State` or `QueryState` or similar for reasons I mentioned above.